helm/ingress-azure/templates/deployment.yaml

```diff
@@ -30,6 +30,34 @@ spec:
       - name: {{ .Chart.Name }}
         image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
         imagePullPolicy: {{ .Values.image.pullPolicy }}
+        readinessProbe:
+          httpGet:
+            path: /health/ready
+        {{- if .Values.kubernetes }}
+        {{- if .Values.kubernetes.healthProbeServicePort }}
+            port: {{ .Values.kubernetes.healthProbeServicePort }}
+        {{ else }}
+            port: 8123
+        {{- end}}
+        {{ else }}
+            port: 8123
```

Comment:
is this twice ?

Reply:
Added a default value for `healthProbeServicePort`  in `values.yaml`
This was if/then/elsed twice as a workaround Go's templates (or perhaps my lack of understanding these)